.github/workflows/code-quality.yml

```diff
@@ -27,15 +27,15 @@ jobs:
           pylint-paths: >
             tests
   black:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Why is Python formatting on ubuntu-22.04, while Python and C++ linting is on ubuntu-latest?

In https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/87/, for example, it's wholesale move to ubuntu-22.04.

The changes here stand out as different to the rest.

Reply:
When we only care about Python or are using the `openassetio-build` container, then the version of Ubuntu doesn't really matter, so I figured those cases can be left as `-latest` for a bit of future-proofing.

I perhaps modified TraitGen without really thinking about this. Not sure which approach is best.  Perhaps better to remove any ambiguity and use 22.04 everywhere, even if it does mean a slightly bigger change next time we version bump?

Reply:
What I tend to do is look for patterns when I'm in code review mode, and this just stood out.

But you sound happy with this as-is for future proofing, and to avoid _more_ work on you, so am resolving.